### DISCUSSION TITLE
Bulk upload forcing answer

### DISCUSSION THREAD

**Discussion by johannSc** [immich-app/immich#3562] [Q&A] [answered]
Hello,

I try to upload all my photos from my synology, and I found a problem.
I don't know why, but when I try to do a recursive upload like this:

`node bin/index.js upload --key XXX --server http://192.168.1.1:2283/api --recursive /Volumes/photo/XX/2021/`

**Immish saw only one sub-directory (always the same).**

But if I specify each sub-directory, immish do the job.

So I wrote a little script, like this:

```
#!/bin/sh

IFS=$'\n'
list=`ls /Volumes/photo/XX/2021/`
for i in $list
do
  node bin/index.js upload --key XXX--server http://192.168.1.1:2283/api --recursive /Volumes/photo/XX/2021/$i
done
```

It's ok, but for each sub-directory immish ask me "_Do you want to start upload now? (y/n)_".

I tried to pipe the node command, but it failed. I read the doc but I hasn't find how to specified --yes, do you know how to do that?

Regards

**Comment by etnoy** [comment, marked as answer]
We're working on a new cli you can try. It's in the cli folder in the main repo

**Reply by johannSc** [reply]
thanks, I will try that.
